src/modules/api/user/user.controller.ts

```diff
@@ -1,7 +1,35 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Get, Param } from '@nestjs/common';
 import { UserService } from './user.service';
+import { IResult } from 'src/common/interfaces/response';
 
-@Controller('users/:id')
+@Controller('users')
 export class UserController {
   constructor(private userService: UserService) {}
```

Comment:
make userService readonly

Reply:
why exactly would we need that ?

Reply:
To enforce that we cant play with userService